Docs: Add section about adding new dependencies to WordPress packages (#16876)
WordPress/gutenberg: packages/README.md

	- Usage example
	- `Code is Poetry` logo (`<br/><br/><p align="center"><img src="https://s.w.org/style/images/codeispoetry.png?1" alt="Code is Poetry." /></p>`)

### Adding New Dependencies

> There are two types of dependencies that you might want to add to one of the existing WordPress package.

Reviewer: ```suggestion
There are two types of dependencies that you might want to add to one of the existing WordPress packages.
```

#### Production Dependencies

> Production dependencies are stored in the `dependencies` section of `package.json` file of the package's folder. The simplest way to add such dependency to one of the packages is to run a very convenient [lerna add](https://github.com/lerna/lerna/tree/master/commands/add#readme) command from the root of the project.

Reviewer: ```suggestion
Production dependencies are stored in the `dependencies` section of the package’s `package.json` file. The simplest way to add such a dependency to one of the packages is to run a very convenient [lerna add](https://github.com/lerna/lerna/tree/master/commands/add#readme) command from the root of the project.
```

_Example:_

```bash
lerna add lodash packages/a11y
```

> This command adds the latest version of `lodash` as a dependency to the `@wordpress/a11y` package which is located in `packages/a11y` folder.

Reviewer: ```suggestion
This command adds the latest version of `lodash` as a dependency to the `@wordpress/a11y` package, which is located in `packages/a11y` folder.
```
